juntagrico_billing/admin/__init__.py

```diff
@@ -23,6 +27,11 @@ class SubscriptionTypeAccountInline(admin.TabularInline):
     extra = 0
 
 
+# add inlines on juntagrico models
+addons.config.register_model_inline(Member, MemberAccountInline)
```

Comment:
Why do we need to register juntagrico admin stuff here? Shouldn't this get registered in `juntagrico.admin`?

Reply:
These are additional inlines that come from this addon.
They were in `juntagricoapps.py` before, which is now obsolete.